GitUI/CommandsDialogs/BrowseDialog/FormBrowseMenus.cs

```diff
@@ -153,7 +153,10 @@ private static void CreateToolStripSubMenus(ToolStrip senderToolStrip, ToolStrip
                     key = toolbarItem.Name;
                 }
 
-                bool visible = LoadVisibilitySetting(key);
+                bool visible = LoadVisibilitySetting(key, IsVisibleByDefault(key));
+
+                bool IsVisibleByDefault(string buttonKey) => !buttonKey.Contains("tsbShowReflog");
```

Comment:
```suggestion
                bool IsVisibleByDefault(string buttonKey) => !buttonKey.Contains(nameof(tsbShowReflog));
```

Reply:
I would have liked (and tried) but it's not possible because that's a private field of another class 😕

Reply:
I will expose the string as an `internal const`. I think it is better than hard-coded...